system/ee/installer/updates/ud_6_01_00.php

```diff
@@ -39,6 +38,15 @@ public function do_update()
         return true;
     }
 
+    private function removeRteExtension()
+    {
+        ee()->db->where('name', 'Rte')->update('fieldtypes', ['version' => '2.0.1']);
+
+        ee()->db->where('module_name', 'Rte')->update('modules', ['module_version' => '2.0.1']);
```

Comment:
If this is the remove function, shouldn't the version be updated to 2.0.0 as part of the rollback?

Reply:
I don't think this qualifies as rollback. Removing extension is part of update, which removes the code that is not needed. Since there is change in DB, the version is being updated